docs/topics/templates.txt

```diff
@@ -401,6 +401,40 @@ applications. This generic name was kept for backwards-compatibility.
 
   It defaults to the value of :setting:`FILE_CHARSET`.
 
+.. versionadded:: 1.9
+
+    The ``libraries`` and ``builtins`` arguments were added.
+
+.. _django-template-backend-libraries:
```

Comment:
I prefer to avoid linking to text without a heading. Could we say something like "use the `'libraries'` and `'builtins'` DjangoTemplates OPTIONS."